.jscsrc

```diff
@@ -38,7 +38,7 @@
    "disallowYodaConditions": true,
    "maximumLineLength": 120,
    "requireBlocksOnNewline": true,
-   "requireCamelCaseOrUpperCaseIdentifiers": true,
+   "requireCamelCaseOrUpperCaseIdentifiers": false,
```

Comment:
What are the exceptions in our naming conventions to disable this? Just wondering.